react.js

```diff
@@ -0,0 +1,3 @@
+console.log('Warning: This module is deprecated and will be removed in future versions. Please use the module from ./vendors/react instead.');
```

Comment:
```suggestion
console.log('Warning: This module is deprecated and will be removed in future versions. Please use the module exposed via "camunda-modeler-plugin-helpers/vendors/react.js" instead.');
```

Reply:
Let's use this with extension, and use the full export.

Reply:
Also, consider to log this as an error, so a clear stack trace is provided.

Reply:
Given that this change is major breaking anyway I don't see a strong reason to include this change.